inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl

```diff
@@ -173,7 +173,7 @@
             {% do viewInDisplayed.append(1) %}
           {% endif %}
           {{ comma() }}
-          <a href='{{ actual_url }}'>{{ url.get('description') | weblinks }}</a>
```

Comment:
This is not correct... what's wrong is the translation at display time, but it's ok to have some logic that displays the right thing.